apps/pip/dev/base/kustomization.yaml

```diff
@@ -4,6 +4,7 @@ resources:
   - ../../../base # This is apps/base instead of apps/pip/base as preview doesn't install everything
   - ../../identity/pip-azure-identity.yaml
   - ../../../rbac/dev-role.yaml
+  - ../../data-management/pip-data-management.yaml
```

Comment:
Can you remove this from dev? I can only see Pip PR releases in dev so this doesn't look needed